ci/test_case

```diff
@@ -41,48 +41,61 @@ _test_case=$(
 # |      gemma-7b |   √  |   √  |   ×  |   ×  |   ×  |   × |  32  |  32  |
```

Comment:
Should we add the "-kvd" parameter to this table for better understanding of our test case coverage?

Reply:
done